RELEASE/scripts/autoscend.ash

```diff
@@ -902,7 +902,7 @@ boolean fortuneCookieEvent()
 			goal = $location[The Castle in the Clouds in the Sky (Top Floor)];
 		}
 
-		if (goal == $location[The Castle in the Clouds in the Sky (Top Floor)] && (get_property("semirareLocation") == goal || item_amount($item[Mick\'s IcyVapoHotness Inhaler]) > 0 || internalQuestStatus("questL10Garbage") < 9 || get_property("lastCastleTopUnlock").to_int() < my_ascensions() || get_property("sidequestNunsCompleted") != "none" || in_koe()))
```

Comment:
shouldn't this be going into the 
`goal = $location[The Castle in the Clouds in the Sky (Top Floor)];`
because that is where the inhaler is found.

if I understand what you are doing, your logic is that after we set to get the inhaler if we get the inhaler then we force it to limrick if there is some failstate and... holy magicarp this is how ALL our logic for semirares is done!
this is aweful! we absolutely need to refactor the whole thing. this is a really backwards way of selecting an action

well in this case your addition is actually correct and inline with the current standard code handling. I guess we can go ahead and merge it for now. a complete refactor could be done later.